lib/std/json.zig

```diff
@@ -1990,7 +1990,7 @@ pub const Parser = struct {
             .Some => {
                 const output = try allocator.alloc(u8, s.decodedLength());
                 errdefer allocator.free(output);
-                try unescapeValidString(output, slice);
+                if (output.len == s.decodedLength()) try unescapeValidString(output, slice) else return error.LengthMismatch;
```

Comment:
Ditto.

Reply:
i will remove this as well

Reply:
made change here https://github.com/ziglang/zig/pull/14370/commits/91701aac12d6c24de50892b8c1e16973d641b483